netbox/dcim/views.py

```diff
@@ -2610,8 +2671,9 @@ def get_extra_context(self, request, instance):
 
         return {
             'vdc_table': vdc_table,
-            'bridge_interfaces_table': bridge_interfaces_tables,
-            'child_interfaces_table': child_interfaces_tables,
+            'bridge_interfaces_table': bridge_interfaces_table,
+            'child_interfaces_table': child_interfaces_table,
+            'mac_addresses_table': mac_addresses_table,
```

Comment:
This isn't needed if we're using an embedded table in the interface view template; the table is populated using a separate HTTP request.